riotfile.py

```diff
@@ -81,6 +81,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         "hypothesis": latest,
     },
     venvs=[
+        Venv(
+            pys=["3"],
+            name="autohook",
+            command="hooks/autohook.sh install",
```

Comment:
we can just document this step in our README, we don't need to run it in riot/venv do we?

Reply:
Definitely not necessary to be in riot, open to removing it and just adding to the README if that's the prevailing opinion 😄

Reply:
@brettlangdon updated the README and removed the riot command, let me know if that looks good